tests/test_catalogMatch.py

```diff
@@ -32,6 +32,8 @@
 from lsst.daf.butler import DatasetRef, DatasetType, DimensionUniverse, StorageClass
 from lsst.meas.algorithms import ReferenceObjectLoader
 
+rng = np.random.default_rng(12345)
```

Comment:
Since you are no longer relying on modifying global numpy random behavior and are using an instance of a random generator, can you move this to be an instance variable of the class you use it in?